indico/modules/events/timetable/operations.py

```diff
@@ -118,3 +121,57 @@ def fit_session_block_entry(entry, log=True):
         entry.event_new.log(EventLogRealm.management, EventLogKind.change, 'Timetable',
                             "Session block fitted to contents", session.user,
                             data={'Session block': entry.session_block.full_title})
+
+
+def move_timetable_entry(entry, parent=None, day=None):
+    """Move the `entry` to another session or top-level timetable
+
+    :param entry: `TimetableEntry` to be moved
+    :param parent: If specified then the entry will be set as a child
+                         of parent
+    :param day: If specified then the entry will be moved to the
+                        top-level timetable on this day
+    """
+    if bool(parent) + bool(day) != 1:
+        raise Exception("Wrong number of arguments")
+
+    from indico.modules.events.contributions.operations import update_contribution
+
+    updates = {}
+    contrib_update_data = {}
+    if day:
+        new_start_dt = entry.start_dt.replace(day=day.day, month=day.month)
+        updates['start_dt'] = new_start_dt.astimezone(entry.event_new.tzinfo)
```

Comment:
I don't think you need `.astimezone` here since we are just passing it to `update_timetable_entry` and it's going to be converted back to UTC.